build.sbt

```diff
@@ -190,8 +192,23 @@ def jsbridge(binary: String, version: String) =
     name := "mill-js-bridge",
     libraryDependencies ++= Seq("org.scala-js" %% "scalajs-tools" % version)
   )
+
 lazy val scalajsbridge_0_6 = jsbridge("0.6", "0.6.21")
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.scala-js" %% "scalajs-sbt-test-adapter" % "0.6.21",
+      "org.scala-js" %% "scalajs-test-interface"   % "0.6.21"
```

Comment:
`scalajs-test-interface` is a JS-only artifact. It shouldn't be part of the dependencies of the *bridge*, which, IIUC, is a JVM project.

`scalajs-test-interface` should be added as a dependency of the JS projects being compiled/tested, however (next to `scalajs-library`, basically, but only in the Test configuration, or its equivalent concept in Mill).

Reply:
Thank you, fixed that one